src/main/java/org/breedinginsight/brapps/importer/model/base/Germplasm.java

```diff
@@ -192,6 +197,23 @@ public BrAPIGermplasm constructBrAPIGermplasm(BreedingMethodEntity breedingMetho
             germplasm.putAdditionalInfoItem("breedingMethod", breedingMethod.getName());
         }
 
+        // Synonyms
+        String blobSynonyms = getSynonyms();
+        List<BrAPIGermplasmSynonyms> brapiSynonyms = new ArrayList<>();
+        if (synonyms != null) {
+            List<String> synonyms = Arrays.asList(blobSynonyms.split(";")).stream()
+                    .map(synonym -> synonym.strip())
+                    .distinct()
```

Comment:
Is silently dropping duplicates what is desired? Also synonyms are currently only removed as duplicate if case is the same. Is that what is wanted? Can also have the same synonyms for different germplasm (not including the key), not sure if that matters.

Reply:
Relaying these questions to @shawnyarnes